docs/clients/brushless_drive.rst

```diff
@@ -105,121 +105,121 @@ Message Table
++--------+-------------------------------+----------------+-----------+----------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| 6      | drive_coast                   | set            |           |          | Disables all drive circuitry, motor passively coasts                                                                                                                                                                                                                                                                                                                                                        |
++--------+-------------------------------+----------------+-----------+----------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| 7      | drive_angle_offset            | get            | float     | rad      | Analogous to motor timing. This is internally computed by the motor.                                                                                                                                                                                                                                                                                                                                        |
++--------+-------------------------------+----------------+-----------+----------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| 8      | drive_pwm                     | get            | float     | pwm      | The applied pwm after all computation and limiting [-1, 1]                                                                                                                                                                                                                                                                                                                                                  |
```

Comment:
I think pwm should generally be PWM wherever it shows up in here when it's not a parameter name